lib/authzHttpClient.js

```diff
@@ -45,7 +45,12 @@ async function _handleRequest(target, thisArg, args, oAuth2Client) {
 
   // create the authz header using the token
   // add the authz header to the headers object of args
-  const [url, options] = args;
+  let options = {};
+
+  const [url] = args;
+  if(args[1]) {
+    options = args[1];
+  }
```

Comment:
Can we just use defaults? Like this:

```js
const [url, options = {}] = args;
```

Reply:
That looks much better, updated here: https://github.com/digitalbazaar/bedrock-oauth2-client/pull/10